examples/cfd/08_channel_flow.ipynb

```diff
@@ -0,0 +1,541 @@
+      "text/plain": [
+       "<Figure size 1100x700 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#NBVAL_IGNORE_OUTPUT\n",
+    "fig = pyplot.figure(figsize = (11,7), dpi=100)\n",
+    "pyplot.quiver(X, Y, u, v);"
```

Comment:
Can we have the contour of the pressure as well here for comparison with the devito result.